src/ucp/wireup/wireup.c

```diff
@@ -290,8 +289,12 @@ ucp_wireup_process_request(ucp_worker_h worker, const ucp_wireup_msg_t *msg,
         /* wireup request for a specific ep */
         ep = ucp_worker_get_ep_by_ptr(worker, msg->dest_ep_ptr);
         ucp_ep_update_dest_ep_ptr(ep, msg->src_ep_ptr);
-        ucp_ep_flush_state_reset(ep);
-
+        if (!(ep->flags & UCP_EP_FLAG_LISTENER) &&
+            ucp_ep_config(ep)->p2p_lanes) {
```

Comment:
pls add comment why no need to reset flush state in this case, and remove empty "else"

Reply:
https://github.com/openucx/ucx/pull/2889/commits/a5d378bb140f76bc1f2d110ded795635b5fef866#diff-5e97c1d955a563dd705187d5ded3b527R294